docs/src/services-details.md

```diff
@@ -0,0 +1,62 @@
+## Services Details
```

Comment:
we can add a link to this page in https://github.com/developmentseed/eoAPI/blob/main/docs/mkdocs.yml#L20-L22

Reply:
done ✅